.github/workflows/release-sass-api.yml

```diff
@@ -0,0 +1,33 @@
+# This has to be a separate workflow to satisfy pub.dev's somewhat draconian
+# requirements for when and how GitHub actions are allowed publish packages.
+# Specifically, it requires that each publish action be triggered by a tag that
+# contains the version number for that action, so we can't just publish sass_api
+# as part of the standard sass release flow because that was triggered by a tag
+# with the sass version number.
+name: Release sass-api
+
+on:
+  push:
+    tags: ['sass-api-[0-9]+.[0-9]+.*']
+
+jobs:
+  deploy_sass_api:
+    if: github.event.repository.fork == false
+    permissions:
+      attestations: write
+      contents: write
+      id-token: write
```

Comment:
`contents` allows create commit/tag/release, `id-token` and `attestations` are needed for artifact attestations. I don't think you need any of these here.